kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/BuiltinLibrary.java

```diff
@@ -382,7 +382,7 @@ private void addItem(String name, String fxmlText, String section, String iconNa
     }
     
     
-    private static String makeFxmlText(Class<?> componentClass) {
+    public static String makeFxmlText(Class<?> componentClass) {
```

Comment:
Move this method up and place it along with other public methods. If possible, add some Javadoc.

Reply:
Done